jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/JKubeProjectUtilTest.java

```diff
@@ -26,14 +27,27 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 class JKubeProjectUtilTest {
 
   @TempDir
   File temporaryFolder;
 
+  private JavaProject project;
+
+  private File artifactFile;
+
+  private boolean artifactFileCreated;
+
+  private File finalOutputArtifact;
+
+  @BeforeEach
+  void setUp() throws IOException {
+    project = JavaProject.builder().build();
+    artifactFile = new File(temporaryFolder, "foo-test-1.0.0.jar");
+    artifactFileCreated = artifactFile.createNewFile();
```

Comment:
Could you please use `Files.createFile(temporaryFolder.toPath().resolve("foo-test-1.0.0.jar"))` here instead? It would automatically throw [FileAlreadyExistsException](https://docs.oracle.com/javase/9/docs/api/java/nio/file/FileAlreadyExistsException.html) when file already exists (you won't need to check for return value like you're doing here)